addons/web/static/src/legacy/js/views/qweb/qweb_view.js

```diff
@@ -1,207 +0,0 @@
-            // move control panel buttons from the view to the control panel
-            // area
-            var $cp_buttons = self.renderer.$('nav.o_qweb_cp_buttons');
-            $cp_buttons.children().appendTo(self.$buttons.empty());
-            $cp_buttons.remove();
-        });
-    },
-
-    //--------------------------------------------------------------------------
-    // Handlers
-    //--------------------------------------------------------------------------
-
-    /**
-     * Lazy toggle. Content is not remembered between unfolds.
-     */
-    _onLazyToggle: function (e) {
-        // TODO: add support for view (possibly action as well?)
-        var $target = $(e.target);
-        var $t = $target.closest('[data-model]');
-        if (!($target.hasClass('fa-caret-down') || $target.hasClass('fa-caret-right'))) {
-            $target = $t.find('.fa-caret-down, .fa-caret-right');
-        }
-
-        var data = $t.data();
-        if (this._fold($t)) {
-            $target.removeClass('fa-caret-down').addClass('fa-caret-right');
-            return;
-        }
-
-        // NB: $.data() automatically parses json attributes, but does not
-        //     automatically parse lone float literals in data-*, so a
-        //     data-args (as a json object) is very convenient
-        var args = data.args || _.omit(data, 'model', 'method', 'id');
-
-        return this._rpc({
-            model: data.model,
-            method: data.method,
-            args: data.id ? [data.id] : undefined,
-            kwargs: args // FIXME: context?
-        }).then(function (s) {
-            return $(s);
-        }).then(function ($newcontent) {
-            $t.data('children', $newcontent).after($newcontent);
-            $target.removeClass('fa-caret-right').addClass('fa-caret-down');
-        });
-    },
-    /**
-     * Attempts to fold the parameter, returns whether that happened.
-     */
-    _fold: function ($el) {
-        var $children = $el.data('children');
-        if (!$children) {
-            return false;
-        }
-
-        var self = this;
-        $children.each(function (_i, e) {
-            self._fold($(e));
-        }).remove();
-        $el.removeData('children');
-        return true;
-    }
-});
-
-/**
- * view
- */
-var QWebView = AbstractView.extend({
-    display_name: _lt('Freedom View'),
-    icon: 'fa-file-picture-o',
-    viewType: 'qweb',
-    // groupable?
-    enableTimeRangeMenu: true,
-    config: _.extend({}, AbstractView.prototype.config, {
-        Model: Model,
-        Renderer: Renderer,
-        Controller: Controller,
-    }),
-
-    /**
-     * init method
-     */
-    init: function (viewInfo, params) {
-        this._super.apply(this, arguments);
-        this.loadParams.viewId = viewInfo.view_id;
-    }
-});
-
-registry.add('qweb', QWebView);
-export default {
-    View: QWebView,
-    Controller: Controller,
-    Renderer: Renderer,
-    Model: Model
-};
```

Comment:
you should clean up server side stuff as well. See the [commit](https://github.com/odoo/odoo/commit/58e0c88eced6) that introduces the qweb view.